### ISSUE TITLE
autoResize: true isn't setting the height and width correctly with Vue shape nodes

### ISSUE THREAD

**Issue by harshaktg** [antvis/X6#3060]
### Describe the bug

I have created an app with x6-vue-shape, where in when you click on the node everytime it increases the height by 100px. I have noticed that at some point the graph is getting cut. When inspecting, it looks like the height is small, and it isn't accommodating the entire svg.

### Your Example Website or App

NA

### Steps to Reproduce the Bug or Issue

1, Create an X6 graph with full x6-vue-shape nodes
2. Add an event to the node where for every click on it, it increases the height of the node by 100px
3, At one point of time, the graph will get cut.

### Expected behavior

Ideally the graph should compute the height and width perfectly after the rending is fully complete. I suppose it is calculating the height before rendering.

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]

**Comment by NewByVector** [comment]
What version of x6 are you using and do you have a reproducible demo.

**Comment by harshaktg** [comment]
I tried using both v1.x and also tried v2.x. In both the places there was no resolutions. 
I don't have a demoable version.

Basically it about a few nodes on the left and right (dagree layout) built from x6-vue-shape. On click of the node, it expands it and append 5 entries as its port. (this in turn increases the size of the node). When there are multiple nodes in the same level, we do add a manual function to adjust the position( y coordinate) of the below node, based on the height of the previous one. After all these computations, the autoResize does it's function. And it gets cut off
<img width="401" alt="image" src="https://user-images.githubusercontent.com/40317596/208304331-39710d0c-745f-4d3f-940d-6e491cf6bfee.png">
<img width="804" alt="image" src="https://user-images.githubusercontent.com/40317596/208304377-9c322d25-5306-4505-9136-f4292c40ff1a.png">

I have tried manually increasing the height inorder to check if it is a height issue. Tried increasing it and it looks fine
<img width="942" alt="image" src="https://user-images.githubusercontent.com/40317596/208304445-263ec14a-a837-4823-ba24-a6c6fd0f5d12.png">
